### DISCUSSION TITLE
Laut listeners not appearing on Azura

### DISCUSSION THREAD

**Discussion by flashradio** [AzuraCast/AzuraCast#5882] [General]
After the last update on Azuracast tree day ago, the laut listener script doesn't work. Could someone tell me if there is a solution or that feature doesn't work anymore.
Thank you

**Comment by BusterNeece** [comment]
@flashradio The laut listener script? Can you link that?

**Comment by flashradio** [comment]
Yes. This one been working till the last Azuracast update
https://flashradio.stream.laut.fm:8443//status-json.xsl?mount=/flashradio

**Comment by BusterNeece** [comment]
@flashradio And you have that set up as a remote relay? You may want to fix that double `/`, and also by specifying it's Icecast, we don't need the `status-json.xsl?mount=/flashradio` part, as long as you provide the `/flashradio` mount.

**Reply by flashradio** [reply]
It is not a remote relay. It is a internal script that you can see the listener of laut on azura till now

**Reply by BusterNeece** [reply]
@flashradio I'm not familiar with the script, I need to know more about it. We don't directly consume data from laut.fm in AzuraCast itself, unless you've added a script or set it up as a remote relay.

**Reply by flashradio** [reply]
The code goes on "Relays" but only to see listeners not to relay the Azuracast signal on Laut. I dont if now if this only only works when the Azuracast signal it is broadcasting  live on Laut

**Reply by flashradio** [reply]
It is not a repeater but a relay without auto DJ

**Reply by BusterNeece** [reply]
@flashradio Without being able to test it directly with your credentials I don't know what else to say. You can debug the Now Playing run for any given station by looking at the bottom of the System Debugger page, selecting the station and then selecting the Now Playing task.

**Comment by flashradio** [comment]
Does not work neither

**Reply by BusterNeece** [reply]
You're not giving us enough log information to know what's going on. We can't help unless you post specific error messages, including the ones that may appear during that Now Playing debug step.

**Comment by flashradio** [comment]
What do you mean? I dont have any error log. The thing is that before the
last update I all see the listener from laut in the profile page, with the
script I all show in the begining of this post, and after the update that
feature is gone after the last update and from that time I can not see the
listener from laut. I supposed somebody could help me, but I see its
difficult even to understand what I mean, so no problem, its ok.

**Comment by flashradio** [comment]
In the laut.fm forum they gave me that script to insert it in the
"Repeater" section of Azura, a year ago, so I can see my audience in my
laut station also.  Until the last Azura update a few days ago. In laut.fm
they tell me that they haven't changed anything, so they told me to comment
it here in case somebody knew something about why it doesn't work now. But
it has nothing to do with "Playing Now". But thanks anyway.

**Comment by BusterNeece** [comment]
I'm pushing an update that may fix this; it should be deployed in the next 15 or so minutes; try updating once it's finished and see if it resolves this issue for you.

**Comment by flashradio** [comment]
OK thank you. If it is not resolved, there is no problem, as this incident
does not affect the operation of Azura.

**Reply by BusterNeece** [reply]
Try updating now and let me know how it goes. I changed how URLs are processed.

**Comment by flashradio** [comment]
Sorry but where is the update button? When I open the administration page I
cant find it

**Reply by BusterNeece** [reply]
https://docs.azuracast.com/en/getting-started/updates

**Comment by flashradio** [comment]
Thanks, I'll check it out. But Do you think it could cause some other
problem and I would have to start over, is there such a possibility?
Because I have more than 9,000 songs, jingles, etc

**Reply by BusterNeece** [reply]
I'm not going to try to convince you to update! I'm asking you to verify whether the latest update fixed things. If you're unsure if it will cause problems, I'd recommend taking a backup first.

**Comment by flashradio** [comment]
Thanks, but for now I'm going to think about it, because even if I make the
backup, I would have to reassign all the songs and so on and it takes a
long time, even months. I'll think about it a bit before making a decision.
Thank you very much, sincerely for your help.

**Reply by BusterNeece** [reply]
No, if you take a backup, you can just restore from that and have all your playlist assignments set up.

It would NOT take months. Maybe a few minutes at most.

I need you to update to confirm whether this issue is fixed or not before we tag a new stable version, but if you're unwilling to do that, then I'll just close this issue.

**Comment by flashradio** [comment]
Ok, I will do that. But not with backup cause I have minimum 50 gb in
files, so take hours for have it.
whatever god wants.🙂

**Comment by flashradio** [comment]
I've already updated, but for now it's still the same

**Reply by BusterNeece** [reply]
I don't know that I can provide any further assistance without logs from the Now Playing sync task. Sorry about that.

**Comment by flashradio** [comment]
No problem, I understand.
Thanks very much

**Comment by RadioHerzblut** [comment]
It is currently not possible to query the stream. Hope there is a solution here again ;-)
